analytic_tag_dimension/models/analytic_dimension_line.py

```diff
@@ -10,6 +11,32 @@ class AnalyticDimensionLine(models.AbstractModel):
     _description = "Analytic Dimension Line"
     _analytic_tag_field_name = "analytic_tag_ids"
 
+    @api.model
+    def get_model_names(self):
+        return [
+            "account.move.line",
+            "account.analytic.line",
+            "account.invoice.report",
+        ]
+
+    def _reflect(self):
+        super()._reflect()
+        if (
+            self._inherit != self._name
+            and self._inherit in self.get_model_names()
+            and self.env.context.get("check_fields", True)
+        ):
+            fields_inherit = self.env["ir.model.fields"].search(
+                [
+                    ("model_id.model", "=", self._inherit),
+                    ("name", "not in", self.fields_get_keys()),
+                ]
+            )
+            if fields_inherit:
+                model = self.env["ir.model"].search([("model", "=", self._name)])
```

Comment:
Why do you need to search again the model if you already have it on any of the fields_inherit records?

Reply:
https://github.com/OCA/account-analytic/pull/379#discussion_r634997336

Reply:
I still don't understand how copying a field is the solution. This duplicates the record.